anndata/_core/index.py

```diff
@@ -105,6 +105,25 @@ def name_idx(i):
         raise IndexError(f"Unknown indexer {indexer!r} of type {type(indexer)}")
 
 
+def _normalize_slice(s: slice, length: int):
```

Comment:
maybe `_clip_slice_to_length`? I'm not sure what "normalize" means.

Reply:
This also remove's any `None` values from the slice. I guess I think of a slice with all elements set as the "normal form" of the slice. 

Numba calls this: `_normalize_slice` or `fix_slice`.